### PR TITLE
Documentation contrib update

### PR DESCRIPTION
**Pull request by fbolton**
In order to capture what we have learned from recent discussions about formatting and conventions in Debezium docs, I have added some contributor notes to the `DOCUMENTATION.md` file. This is still a work in progress, but I thought it would be good to get the ball rolling.

I'm not sure if this is the ideal place to put these notes. If necessary, I could add them somewhere else and create a new PR.

### CODE REVIEW

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -116,4 +118,62 @@ Whenever any release is made, its important that the `_attributes.adoc` file [he
...
+At build time, AsciiDoc reads the value of the standard `imagesdir` attribute to discover the location of images.
+There is no need to set this attribute yourself, it is already defined for you.
+Note that if you view or render a _single_ AsciiDoc file, you will not be able to view the image.
+Because of the way the images are organized (in combination with the `imagesdir` attribute), you can only see the images correctly rendered when you build the whole documentation set using Antora.
+
+### Best practices
+
+Note the following additional recommendations:
+
+* If your contribution also involves reformatting large amounts of text (modifying white space, line breaks, and so on), please make these reformatting updates in a _separate_ commit.
```
Comment (gunnarmorling) [inline DOCUMENTATION.md]: 💯 👍

**Comment by gunnarmorling** [comment]
Applied; thanks a lot, @PaulRMellor and @fbolton!

**Comment by fbolton** [comment]
Thank you, @gunnarmorling ! What a relief to get this merged!

**Comment by Naros** [comment]
Thanks @gunnarmorling for handling this one.

**Pull request merged**